Source/SpacePort/SpacePort/Menu.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SpacePort
+{
+    public class Menu
+    {
```

Comment:
I don't think it's necessary to create an entire class just for the menu. You maybe did that to separate the menu from the program class, perhaps it is okay to do so. But I personally believe that this should be in Program.cs if the menu isn't very complex.

Reply:
It will become somewhat complex, hence we did a separate class :)